From 1ce99798fbb4b2086cbcce67907191ae36aadbb8 Mon Sep 17 00:00:00 2001
From: Todd Whiteman <todd.whiteman@joyent.com>
Date: Tue, 3 Apr 2018 10:46:11 -0700
Subject: [PATCH] TRITON-288 sdcnode updates for March 2018 node.js security
 releases Reviewed by: Marsell Kukuljevic <marsell@joyent.com> Approved by:
 Marsell Kukuljevic <marsell@joyent.com>

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 2af1711..e1823f8 100644
--- a/Makefile
+++ b/Makefile
@@ -50,7 +50,7 @@ CLEAN_FILES	+= node_modules cscope.files
 
 # The prebuilt sdcnode version we want. See
 # "tools/mk/Makefile.node_prebuilt.targ" for details.
-NODE_PREBUILT_VERSION=v4.8.7
+NODE_PREBUILT_VERSION=v4.9.0
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_IMAGE=18b094b0-eb01-11e5-80c1-175dac7ddf02
 	NODE_PREBUILT_TAG=zone
-- 
2.21.0

